tmt/steps/provision/artemis.py

```diff
@@ -285,7 +289,8 @@ def _create(self) -> None:
                 },
             'os': {
                 'compose': self.image
-                }
+                },
+            'kickstart': self.kickstart or {}
```

Comment:
I'm afraid the `kickstart` field needs to be added only when the API version allows it. It seems to be the first case like this, but it's similar to how `artemis-cli` does it: check `self.api_version`, if it's good enough, add `kickstart` key as you do, if the version is not good enough *and* there's a kickstart (`self.kickstart` is not empty), raise a `ProvisionError`.

Reply:
Yeah, you are right. I added the version check